bin/update-schematron.php

```diff
@@ -3,15 +3,22 @@
 
 declare(strict_types=1);
 
+use GitWrapper\GitWrapper;
 use Symfony\Component\Filesystem\Filesystem;
 
 require_once __DIR__.'/../vendor/autoload.php';
 
 $filesystem = new Filesystem();
-
-$target = __DIR__.'/../lib/schematron';
-
-$base = 'https://github.com/Schematron/schematron/raw/master';
+$git = new GitWrapper();
+$git->streamOutput();
+
+$repoUri = 'https://github.com/Schematron/schematron';
+$repoDir = __DIR__.'/../tmp/schematron';
+$patchesDir = __DIR__.'/../tmp/patches';
+$targetDir = __DIR__.'/../lib/schematron';
+$patches = [
```

Comment:
short comment to include on what the patch resolves without having to open the related PR page?